src/com/yetanalytics/lrs_admin_ui/views/status.cljs

```diff
@@ -0,0 +1,227 @@
+
+(defn title-loading-spinner
+  [vis-key]
+  [:span.vis-title-loading-spinner
+   {:class (if @(subscribe [:status/loading? vis-key])
+             "spinner"
+             "")}])
+
+(defn platform-pie
+  []
+  [:div.vis-pie
+   [:h4 "PLATFORMS"
+    [title-loading-spinner "platform-frequency"]]
+   [vis/pie
+    {:theme (aget vis/theme "material")
+     :data @(subscribe [:status.data/platform-frequency])
+     :labels (fn [c]
+               (let [datum (aget c "datum")
+                     x (aget datum "x")
+                     y (aget datum "y")]
+                 (format "%s: %s"
+                         x y)))}]])
+
+(defn timeline-select-unit
+  []
+  (let [current-unit @(subscribe [:status.params/timeline-unit])]
+    [:div.vis-timeline-controls-select-unit
+     [:label
+      {:for "timeline-select-unit"}
+      "Time Unit"]
+     (into [:select
+            {:id "timeline-select-unit"
+             :value current-unit
+             :on-change #(dispatch [:status/set-timeline-unit
+                                    (fns/ps-event-val %)])}]
+           (for [unit ["year"
+                       "month"
+                       "day"
+                       "hour"
+                       "minute"
+                       "second"]]
+             [:option
+              {:value unit
+               :key (str "unit-" unit)}
+              (cs/capitalize unit)]))]))
+
+(defn- pick-datetime
+  [input-id
+   label
+   value
+   on-change
+   min-datetime
+   & [max-datetime]]
+  [:div.vis-timeline-controls-pick-datetime
+   [:label
+    {:for input-id}
+    label]
+   [:input
+    (cond-> {:id input-id
+             :type "datetime-local"
+             :value value
+             :on-change on-change
+             :min min-datetime}
+      max-datetime (assoc :max max-datetime))]])
+
+(defn timeline-pick-since
+  []
+  [pick-datetime
+   "timeline-pick-since"
+   "Since"
+   @(subscribe [:status.params/timeline-since-local])
+   #(dispatch [:status/set-timeline-since
+               (fns/ps-event-val %)])
+   "1970-01-01T00:00:00"
+   @(subscribe [:status.params/timeline-until-local])])
+
+(defn timeline-pick-until
+  []
+  [pick-datetime
+   "timeline-pick-until"
+   "Until"
+   @(subscribe [:status.params/timeline-until-local])
+   #(dispatch [:status/set-timeline-until
+               (fns/ps-event-val %)])
+   @(subscribe [:status.params/timeline-since-local])])
+
+(defn timeline-controls
+  []
+  [:div.vis-timeline-controls
+   [timeline-select-unit]
+   [timeline-pick-since]
+   [timeline-pick-until]])
+
+(def unit-for
+  "Mapping of timeline bucket time unit to suitable FOR sql substring arg."
+  {"year"   4
+   "month"  7
+   "day"    10
+   "hour"   13
+   "minute" 16
+   "second" 19})
+
+(defn- x-tick-format
+  [unit js-date]
+  (subs (.toISOString js-date) 0 (get unit-for unit)))
+
+(defn timeline-chart
+  []
+  (let [x-unit @(subscribe [:status.params/timeline-unit])]
+    [:div.vis-timeline-chart
```

Comment:
It would be nice if you could put a border around the timeline chart. Whenever I navigate the page I often scroll onto the timeline view, and I wonder why the page stopped scrolling.

Reply:
The more I think about this, the more I favor just turning zoom off. The focus stuff is still fiddly, and they _already_ have a way to set the domain with the controls above. This goes back to how this feature is not supposed to be a full-fledged analytics environment so I think I'm just removing it.

Reply:
That works too

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/8f128c38558bddb7d2c51aec33a9deb2dbc28f95